### PR TITLE
Add a short shipping forecast

### LINKED ISSUE #1: Add the Shipping Forecast
Add in a file containing the Shipping Forecast

### PR DESCRIPTION
**Pull request by RogerHar**
Closes #1.

### CODE REVIEW

File: shipping-forecast.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Viking, North Uitsere, South Uitsere: Low, 920, Falling quickly. Good, becoming poor. Heavy icing expected later.
```
Comment (MA-Sharpe) [inline shipping-forecast.md]: A bit too much icing for my liking, but I guess it's getting near to Christmas...
```suggestion
Viking, North Uitsere, South Uitsere: Low, 920, Falling quickly. Good, becoming poor. Icing expected later.
```

**Review by MA-Sharpe — CHANGES_REQUESTED** [review]
Although it is Christmas, there is a bit too much icing for my liking.

**Comment by RogerHar** [comment]
Thanks for the review Mike. I've reduced the amount of icing in a second commit.

**Review by MA-Sharpe — APPROVED** [review]
looks good to me.

**Pull request merged**